### PR TITLE
Add pt_BR locale

### PR DESCRIPTION
**Pull request by draivin**

### CODE REVIEW

File: config/locales/client.pt_BR.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# After this file is changed, it needs to be pushed by a maintainer to Transifex:
+#
+#   tx push -s
+#
+# Read more here: https://meta.discourse.org/t/contribute-a-translation-to-discourse/14882
+#
+# To validate this YAML file after you change it, please paste it into
+# http://yamllint.com/
+
+en:
```
Comment (ZogStriP) [inline config/locales/client.pt_BR.yml]: This line has to be `pt_BR:`

File: config/locales/server.pt_BR.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+en:
```
Comment (ZogStriP) [inline config/locales/server.pt_BR.yml]: This line has to be `pt_BR:`

**Comment by draivin** [comment]
Fixed both of these.

**Comment by ZogStriP** [comment]
Thanks 👍

**Pull request merged**